doc/modules/ROOT/pages/common-datasets.adoc

```diff
@@ -92,6 +92,44 @@ The default is to load IMDB dataset as `undirected = True`. If loaded as directe
 |===
 
 
+=== LastFM
+
+A heterogeneous graph that is used to benchmark link prediction models, for example used by MAGNN: Metapath Aggregated Graph Neural Network for Heterogeneous Graph Embedding.
+The data and licensing is from https://grouplens.org/datasets/hetrec-2011/[HetRec 2011]
+The original raw data is from http://www.lastfm.com/[LastFM].
+
+The graph contains User and Artists as nodes, each with a `rawId` field that corresponds to the ids from the raw data.
+The `rawId` could be used, for example, to check the artist names by referring back to the HetRec'11 .dat files.
+
+There are three types of relationships.
+`IS_FRIEND` which represents User-User friendship.
+
+`LISTEN_TO` between User and Artist.
+It has a relationship property `weight` that represents the number of times the user listened to the artist.
+
+`TAGGED` between User and Artist.
+It has a `tagID` property where each `tagId` represents a genre,
+It also has `day`, `month`, `year` properties, as well as an equivalent `timestamp` property converted from the day-month-year,
+which represents when the user tagged that artist with that `tagId`.
+Note that a user can tag the same artist at different times with different `tagId`.
```

Comment:
I suggest to use some markup and more structured prose here to improve readability:

![Screenshot 2023-06-08 at 10 54 26](https://github.com/neo4j/graph-data-science-client/assets/2419675/aa009a84-223c-413a-b1ff-bd586b4d1ade)

```suggestion
There are three types of relationships:

* `IS_FRIEND` exists between `User` nodes, representing user-to-user friendships.
* `LISTEN_TO` exists between `User` and `Artist` nodes, indicating the artists a user listens to.
  * `weight` property represents the number of times the user listened to the artist.
* `TAGGED` exists between `User` and `Artist` nodes, indicating artists that a user has tagged.
  * `tagID` property represents a genre
  * `day`, `month`, and `year` properties represents the time when tagging was created
  * `timestamp` represents the same date as milliseconds since EPOCH

Note that a user can tag the same artist multiple times with different `tagID`.
```

What does tagging mean in real-life terms? I'm not sure my suggestion is immediately intuitive.
Is tagging that a user is "claiming" that an artist belongs to a certain genre?